Misc/NEWS.d/next/Library/2018-08-21-16-20-33.bpo-29620.xxx666.rst

```diff
@@ -0,0 +1,2 @@
+In :func:`~unittest.TestCase.assertWarns`, iterate over a copy of
```

Comment:
Making a copy for iteration is an implementation detail. The user visible change is that `assertWarns()` no longer raises a `RuntimeException` when `__warningregistry__` causes importing a module or a module is imported in other thread.